src/tests/test_utils/common_test_utils/src/test_constants.cpp

```diff
@@ -13,7 +13,7 @@ const char *DEVICE_CPU = "CPU";
 const char *DEVICE_GNA = "GNA";
 const char *DEVICE_GPU = "GPU";
 const char *DEVICE_BATCH = "BATCH";
-const char *DEVICE_KEEMBAY = "VPU";
+const char *DEVICE_KEEMBAY = "NPU";
```

Comment:
Can we rename this to DEVICE_NPU and move it before BATCH? Few other occurrences of this define in OV codebase and functional tests need to be updated as well in vpux-plugin repo.

Reply:
Updated in PR https://github.com/openvinotoolkit/openvino/pull/19004 which will continue this thread